The openssl get_certificate working test isn't cross compile friendly.
Just assume it works ok since we're using a modern OpenSSL version.

Signed-off-by: Gustavo Zacarias <gustavo@zacarias.com.ar>
[Bernd: Rebased for 4.2]
Signed-off-by: Bernd Kuhls <bernd.kuhls@t-online.de>

diff -Nura squid-3.5.1.orig/configure.ac squid-3.5.1/configure.ac
--- squid-3.5.1.orig/configure.ac	2015-01-27 11:09:15.035711981 -0300
+++ squid-3.5.1/configure.ac	2015-01-27 11:09:25.829084482 -0300
@@ -1377,10 +1377,6 @@
       ])
     SQUID_STATE_ROLLBACK(check_SSL_CTX_get0_certificate)
 
-    # check for other specific broken implementations
-    if test "x$missing_SSL_CTX_get0_certificate" = "xyes"; then
-      SQUID_CHECK_OPENSSL_GETCERTIFICATE_WORKS
-    fi
     SQUID_CHECK_OPENSSL_CONST_SSL_METHOD
     SQUID_CHECK_OPENSSL_CONST_CRYPTO_EX_DATA
     SQUID_CHECK_OPENSSL_CONST_SSL_SESSION_CB_ARG
